demos/system-services-monitor/checks/service_check.py

```
@@ -0,0 +1,229 @@
+            if result.returncode != 0 or not result.stdout.strip():
+                return []
+
+            found: List[ErrorCategory] = []
+            text = result.stdout.lower()
+            for category, patterns in _ERROR_PATTERNS.items():
+                if any(p.lower() in text for p in patterns):
+                    found.append(category)
+
+            return found
+
+        except (subprocess.TimeoutExpired, Exception) as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)